features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Shared/Alert.jsx

```diff
@@ -95,7 +95,9 @@ class Alert {
                 Notification.newInstance(
                     {
                         transitionName: 'move-down',
-                        style: { bottom: 0, marginLeft: '2%', position: 'absolute' },
+                        style: {
+                            zIndex: '2000', bottom: 0, marginLeft: '2%', position: 'fixed',
```

Comment:
Can't move the styles to Material UI class mechanism, Because the alert component getting rendered outside the Publisher app root element, MUI Theme context not available to it.